lib/gui/components/drive-info-modal/templates/drive-info-modal.tpl.html

```diff
@@ -0,0 +1,59 @@
+<div class="modal-header">
+  <h4 class="modal-title">
+    <span class="glyphicon glyphicon-{{ ::modal.options.icon }}"></span>
+    <span>{{ ::modal.options.title }}</span>
+  </h4>
+  <button class="close" ng-click="modal.reject()">&times;</button>
+</div>
+
+<div class="modal-body">
+  <div class="modal-text"
+    ng-if="modal.options.header && modal.options.description">
+    <h4>{{ ::modal.options.header }}</h4>
+    <p>{{ ::modal.options.description }}</p>
+  </div>
+  <div class="modal-selected-drive">
+    <div class="modal-selected-drive-description">
+      {{ ::modal.selection.getDrive().description }}
+
+    </div>
+    <div class="modal-selected-drive-name">
+      {{ ::modal.selection.getDrive().name }} - {{ ::modal.selection.getDrive().size | gigabyte | number:1 }} GB
+
+
+      <span class="label label-warning"
```

Comment:
This whole set of labels are being copy-pasted from the drive selector. If we're showing them in two places now, we should extract this logic so it can be re-used between both, otherwise we risk having them out of sync.